### PR TITLE
Add missing pub-sub samples

### PR DESCRIPTION
**Pull request by ace-n**
As requested by @jabubake

### CODE REVIEW

File: pubsub/subscriptions.js [hunk 1]
```diff
@@ -84,6 +84,32 @@ function createSubscription (topicName, subscriptionName) {
 }
 // [END pubsub_create_subscription]
 
+// [START pubsub_subscriber_flow_settings]
+function createFlowControlledSubscription (topicName, subscriptionName, maxInProgress, maxBytes) {
```
Comment (jmdobry) [inline pubsub/subscriptions.js]: I guess the Pub/Sub samples were never updated to hide the method definition?

File: pubsub/subscriptions.js [hunk 2]
```diff
@@ -84,6 +84,32 @@ function createSubscription (topicName, subscriptionName) {
...
+      maxMessages: maxInProgress
+    }
+  })
+    .then((results) => {
+      const subscription = results[0];
+
+      console.log(`Subscription ${subscription.name} created with a maximum of ${maxInProgress} unprocessed messages.`);
+
+      return subscription;
+    });
```
Comment (jmdobry) [inline pubsub/subscriptions.js]: And weren't updated to handle errors.

**Review by jmdobry — APPROVED** [review]

File: pubsub/subscriptions.js (hunk 1 above)
Comment (ace-n) [reply pubsub/subscriptions.js]: Correct - I can address that in a separate PR, if you'd like.

File: pubsub/subscriptions.js (hunk 2 above)
Comment (ace-n) [reply pubsub/subscriptions.js]: Correct - see above comment.

File: pubsub/subscriptions.js (hunk 1 above)
Comment (jmdobry) [reply pubsub/subscriptions.js]: Yes please

**Pull request merged**